docs/data/external-oasis-repos.md

```diff
@@ -123,16 +123,15 @@ Verify that the repository is readable over HTTP:
 
 That should print several lines including some gibberish at the end.
 
-Host a Repository on OASIS
-------------------------
+Hosting a Repository on OASIS
+-----------------------------
 
```

Comment:
A short intro here would be helpful I think, like "In order to host a repository on OASIS, do the following steps:".